FirebaseCore/Sources/Public/FirebaseCore/FIROptions.h

```diff
@@ -32,7 +32,7 @@ NS_SWIFT_NAME(FirebaseOptions)
 
 /**
  * An iOS API key used for authenticating requests from your app, e.g.
- * @"AIzaSyDdVgKwhZl0sTTTLZ7iTmt1r3N2cJLnaDk", used to identify your app to Google servers.
+ * The key must begin with "A" and contain exactly 39 alphanumeric characters, used to identify your app to Google servers.
```

Comment:
Looks good. Thanks! Please wrap lines at 100 and then good to go.

Reply:
Thank you! Done.